### DISCUSSION TITLE
Full text search

### DISCUSSION THREAD

**Discussion by lenski88** [storybookjs/storybook#24793] [General] [answered]
Our react project uses Storybook v7. Now the search is possible in the navigation tree by the name of the components. Is it possible to implement keyword search for component descriptions or their properties in Storybook auto documentation. Let's say the description includes the following text "The component used to select products from the category ". I would like to be able to search by this description as well, not just the name of the component or the history in the tree. Thanks

I tried @toptal/davinci-storybook-full-text-search-addon, but for some reason it didn't work

**Comment by domyen** [comment, marked as answer]
Great idea! It's not currently supported, but is something that would be super useful. 

I'd add a search of the text rendered from the component would also be helpful here.

**Reply by lenski88** [reply]
yes, with a large number of components, it can be quite difficult to find what you need with the current search implementation. Especially for new developers on the project. And unfortunately
I have not seen any plugins implementing this feature. I hope it will be in the new version

**Reply by DawidWraga** [reply]
Yes I was about to create a feature suggestion for the same thing. 

The only plugin available doesn't work and the source code is no longer available on GitHub. 

I'd love to see this added to storybook, is there any plans to implement full text search in v8?

**Comment by rad247** [comment]
We would also love this feature. We've struggled with naming our components, because devs, UXers, copy, POs, and other users know them as different terms (ie pop-up for modal).

We're planning on building a docs page with a search feature for the meantime, but it would be even better if the built-in search were configurable to look-up more than just the component title and path name.

**Comment by amalia-colussi** [comment]
This would be a big improvement for us. We use Storybook to document all aspects of our design system, including that aren't tied to one specific component (e.g., capitalization rules). A better search would mean we'd have much better adoption and usage of our documentation.

**Comment by sandrina-p** [comment]
Any news on this topic? 🙇 This would make all the difference to discover content
